roles/nuage_master/tasks/main.yaml

```diff
@@ -1,7 +1,22 @@
 ---
-- name: Create directory /usr/share/nuage-openshift-monitor
```

Comment:
does it not break backward compatibility for RHEL systems? IMO, we should use " /usr/share/nuage-openshift-monitor" for RHEL and "/var/usr/share/nuage-openshift-monitor" for Atomic. can we not conditionally set this variable and use it?